joomla/joomla-cms
# Fix for issue #13531

Pull Request for Issue #13531 .

### Summary of Changes
- [AND] and [OR] operators were not functioning correctly. Fixed.
- some cleanup of whitespaces in XML file and hiding of rulers when the encapsulated fields are not showing.

### Testing Instructions
Code review (with w=1 because of the whitespace reformat in the xml file) [here](https://github.com/joomla/joomla-cms/pull/13535/files?w=1
)

Test fields where showon with [AND] and [OR] are implemented.
for example:
[AND]: Articles->Options->"Articles" tab
Only when both "Show "Read More" and "Show Title with Read More" are set to "Show" the "Read More Limit" field should be shown.

[OR]: Articles->Options->"Editing Layout" tab
Only when both "Frontend Images and Links" and "Administrator Images and Links" are set to "Hide" then the five fields under them should be hidden.

### Documentation Changes Required
None
